ftw/chameleon/tests/test_cook_warning.py

```diff
@@ -24,7 +27,12 @@ def test_warning_logged_in_eager_mode(self):
                      "Template '{}/templates/foo.pt' was unexpectedly cooked while"
                      " eager loading is enabled.".format(
                          os.path.dirname(__file__)))
-        log.check(log_entry, log_entry)
+
+        # FIXME explain why I need to do this!
```

Comment:
If I understand / remember correctly: the `log.check` method does some kind of assertion. Is that true?

Reply:
Yes, it expects *exactly* these messages to be logged.  So `log.check(log_entry, log_entry)` expects exactly 2 messages to be logged - which is true for Plone 4, but I don't understand why.
 Only one message is logged for Plone 5, but that makes sense to me.